fastembed/sparse/bm25.py

```diff
@@ -223,6 +225,9 @@ def embed(
         )
 
     def _stem(self, tokens: list[str]) -> list[str]:
+        if self.disable_stemmer:
+            return tokens
+
```

Comment:
It disables more functionality that we actually would like to disable

I suggest introducing `language = None` which will set stopwords to an empty set and stemmer to None, but will not affect any other things which are not related to the language

Reply:
`language = None` might be better used as a fallback to English. For disabling stemmer, I think, it might be better to use some special value instead.